### PR TITLE
Update cell-painting

### PR DESCRIPTION
**Pull request by yuenler**

### CODE REVIEW

File: cell-painting/1.optimize/get_best_hyperparameters.ipynb [hunk 1]
```diff
@@ -475,7 +322,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.6.11"
+   "version": "3.7.8"
```
Comment (gwaybio) [inline cell-painting/1.optimize/get_best_hyperparameters.ipynb:325]: I'm wondering why the version changed. We have a conda environment in this repo right?

File: cell-painting/1.optimize/optimize.py [hunk 2]
```diff
@@ -1,43 +0,0 @@
-import subprocess
```
Comment (gwaybio) [inline cell-painting/1.optimize/optimize.py:1]: why are these files deleted? Just making sure this was done on purpose

**Review by gwaygenomics — APPROVED** [review]
A couple comments, but nothing stopping from a merge. Go ahead and merge, then create the Regeneron release, and then mint the Zenodo DOI, then we're good to go! Nice work getting this done
